session/bootstrap_test.go

```diff
@@ -388,19 +388,6 @@ func (s *testBootstrapSuite) TestOldPasswordUpgrade(c *C) {
 	c.Assert(newpwd, Equals, "*0D3CED9BEC10A777AEC23CCC353A8C08A633045E")
 }
 
-func (s *testBootstrapSuite) TestBootstrapInitExpensiveQueryHandle(c *C) {
```

Comment:
why remove this test

Reply:
The `InitExpensiveQueryHandle` function is removed so I don't see the point testing against it. But maybe instead of removing the test, I could just remove `InitExpensiveQueryHandle` invocation and just check the existence of that query handler to avoid regression. Let me add that.

Reply:
Done